backend/tests/integration/conftest.py

```diff
@@ -50,9 +50,9 @@ async def load_infrastructure_schema(db: InfrahubDatabase):
     tmp_schema = branch_schema.duplicate()
 
     for file_name in os.listdir(base_dir):
-        file_path = os.path.join(base_dir, file_name)
+        file_path = Path(base_dir, file_name)
 
-        if file_path.endswith((".yml", ".yaml")):
+        if str(file_path).endswith((".yml", ".yaml")):
```

Comment:
`file_path.suffix in (".yml", ".yaml")`

Reply:
Thank you
I knew there was a better way but I was on the plane without internet connection while I did this PR :/